prow/cmd/gerrit/main.go

```diff
@@ -130,8 +132,7 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.StringVar(&o.tokenPathOverride, "token-path", "", "Force the use of the token in this path, use with gcloud auth print-access-token")
 	fs.IntVar(&o.changeWorkerPoolSize, "change-worker-pool-size", 1, "Number of workers processing changes for each instance.")
 	fs.DurationVar(&o.pushGatewayInterval, "push-gateway-interval", time.Minute, "Interval at which prometheus metrics for disk space are pushed.")
-	fs.UintVar(&o.instanceConcurrencyLimit, "instance-concurrency-limit", 5, "Number of concurrent calls that can be made to any single Gerrit host instance simultaneously.")
```

Comment:
For flag safety it might be better to make this a do-nothing first, then delete on a later change. (Fewer issues with rollback/deployment imo)

Reply:
We'll need to replace the usage of this flag with the new flags to avoid getting 429s so we'll have a problem either way if we bump this without updating the flags. I'm not sure if it is preferable to have the binary fail obviously due to a deprecated flag or struggle less obviously due to an omitted flag. :thinking: 
I'll add back the flag for now though.